project/LinkerdBuild.scala

```diff
@@ -529,6 +521,7 @@ object LinkerdBuild extends Base {
 
   // Unified documentation via the sbt-unidoc plugin
   val all = project("all", file("."))
+    .settings(unidocSettings)
     .aggregate(
       admin,
       configCore,
```

Comment:
Should the aggregate list for the all project include etcd? It seems to include everything else defined in this file. And I don't think we'll run tests for etcd by default if it's omitted.